src/github.com/getlantern/flashlight/config/fetcher.go

```diff
@@ -49,8 +48,8 @@ type UserConfig interface {
 
 // NewFetcher creates a new configuration fetcher with the specified
 // interface for obtaining the user ID and token if those are populated.
-func NewFetcher(conf UserConfig, httpFetcher util.HTTPFetcher) Fetcher {
-	return &fetcher{lastCloudConfigETag: map[string]string{}, user: conf, httpFetcher: httpFetcher}
+func NewFetcher(conf UserConfig, httpClient *http.Client) Fetcher {
```

Comment:
All looking great @oxtoacart just wondering on this change. The interface makes it more testable, and hey the [Go best practices post](https://peter.bourgon.org/go-best-practices-2016/) even explicitly recommends it!